website/docs/developer-docs/docs/writing-documentation.md

```diff
@@ -4,60 +4,110 @@ title: Writing documentation
 
 Writing documentation for authentik is a great way for both new and experienced users to improve and contribute to the project. We appreciate contributions to our documentation; everything from fixing a typo to adding additional content to writing a completely new topic.
 
+## Guidelines
+
+To test how the documentation renders you can build locally and then use the Netlify Deploy Preview, especially when using Docusaurus-specific features.
 Adhering to the following guidelines will help us get your PRs merged much easier and faster, with fewer edits needed.
 
 - Ideally, when you are making contributions to the documentation, you should fork and clone our repo, then [build it locally](#set-up-your-local-build), so that you can test the docs and run the required linting and spell checkers before pushing your PR. While you can do much of the writing and editing within the GitHub UI, you cannot run the required linters from the GitHub UI.
 
-- Please refer to our [Style Guide](./style-guide.mdx) for authentik documentation. Here you will learn important guidelines about not capitalizing authentik, how we format our titles and headers, and much more.
```

Comment:
Let's leave this here. I see it was moved to be under the Formatting section, but I'd like to have the Style Guide mentioned early in this page; it applies to more than just formatting.